paper.md

```diff
@@ -24,6 +24,7 @@ authors:
    orcid: 0000-0003-2917-3450
    affiliation: 2
  - name: Jason Gors  # 60 commits, issues: opened 2 participated in 1
+   affiliation: FIXME
```

Comment:
Is having affiliation strictly required? Would be very non inclusive from joss to forbid citizen scientists from publishing

Reply:
That is not my concern. My concern is that without an affiliation declaration they all get `1`, which is Dartmouth College ATM. If that is OK, or even intentional, I suggest to make that explicit, rather than rely on that implicit behavior.

Screenshot without FIXME:

![image](https://user-images.githubusercontent.com/136479/115115027-744a4c80-9f92-11eb-993e-e9c6d7fe6a1c.png)

Reply:
ah, nicely spotted -- I have missed that and it was not intentional.